bigbone/src/main/kotlin/social/bigbone/api/method/AnnouncementMethods.kt

```diff
@@ -0,0 +1,79 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.Pageable
+import social.bigbone.api.entity.Announcement
+import social.bigbone.api.exception.BigBoneRequestException
+
+/**
+ * Allows access to API methods with endpoints having an "api/vX/announcements" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/notifications/">Mastodon notifications API methods</a>
+ */
+class AnnouncementMethods(private val client: MastodonClient) {
+
+    private val announcementsEndpoint = "/api/v1/announcements"
+
+    /**
+     * See all currently active announcements set by admins.
+     * @param withDismissed Boolean If true, response will include announcements dismissed by the user. Defaults to false.
+     * @see <a href="https://docs.joinmastodon.org/methods/announcements/#get">Mastodon API documentation: methods/announcements/#get</a>
+     */
+    fun getAllAnnouncements(
```

Comment:
Please add the `@JvmOverloads` annotation here. Otherwise, when called from Java, default arguments may not be interpreted correctly or you won’t be able to call this method either with or without the Boolean argument.

```suggestion
    @JvmOverloads
    fun getAllAnnouncements(
```

Reply:
I _think_ it’s only necessary for cases where you call `client.performAction` directly.

Right @andregasser?